src/components/file-browser/DownloadFileCommand.ts

```diff
@@ -48,7 +48,7 @@ export const DownloadFileCommand = (props: DownloadFileCommandProps) => {
           children: undefined,
           disabled: !downloadCommand,
           style: { marginLeft: '1ch' },
-          text: downloadCommand,
+          text: downloadCommand || ' ',
```

Comment:
This same defaulting pattern is used above, so I thought it would be fine here, too.